.github/workflows/tauri.yaml

```
@@ -98,3 +98,18 @@ jobs:
+
+      # check for npm package blacklists pkgs across all tauri related packages
+      - uses: rainlanguage/github-chore/•github/actions/npm-blacklist@main
+
+      - uses: rainlanguage/github-chore/•github/actions/npm-blacklist@main
+        with:
+          working-directory: packages/orderbook
+
+      - uses: rainlanguage/github-chore/•github/actions/npm-blacklist@main
+        with:
+          working-directory: packages/ui-components
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)